cli/codegen/generate_code.go

```diff
@@ -0,0 +1,162 @@
+package main
```

Comment:
`cli/codegen/generate_string_constants.go` or smth like this?

Now I don't understand what code is generated. I suggest removing `_code` suffix because file is placed in `codegen` directory and it should be obvious that `generate_*` generates some code.